docs/index.rst

```diff
@@ -1,172 +0,0 @@
-.. sisl documentation master file, created by
```

Comment:
I guess this is an accident, we really need this file ;), please undelete it

Reply:
Yes, I had it in .gitignore by mistake.